src/iam/iam-base.ts

```diff
@@ -429,7 +429,8 @@ export class IAMBase {
         const connection = await Promise.race<NatsConnection | undefined>([
           connect({
             servers: `${protocol}://${this._natsServerUrl}`,
-            timeout
+            timeout,
+            pingInterval: 50 * 1000
```

Comment:
Probably not a big deal, but you mentioned 90 seconds in your description of the change and this seems to be 50 seconds. Is that okay/intentional?

Reply:
Yeah, sorry it was 90 seconds, but I've noticed that this ping interval is not always 90 seconds exactly, it may be longer, so I've changed it to more secure value :)